crypto/sr25519/encoding.go

```diff
@@ -1,23 +1,13 @@
 package sr25519
 
-import (
-	"github.com/tendermint/tendermint/crypto"
-	tmjson "github.com/tendermint/tendermint/libs/json"
-)
-
-var _ crypto.PrivKey = PrivKey{}
```

Comment:
ditto

Reply:
whoops looked at the next file

Reply:
Moved to pubkey.go/privkey.go since that's where the types are defined.